### PR TITLE
[KOGITO-1946] - Release Kogito Operator/CLI 0.11.0

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/.default_config [hunk 1]
```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=latest
```
Comment (sutaakar) [inline test/.default_config:1]: Why is this changed?

Comment (spolti) [reply test/.default_config:1]: should we keep latest for specific version tag?:

Comment (sutaakar) [reply test/.default_config:1]: I think we can keep the latest here, I guess this config is not used for testing of specific release version. @radtriste is it so?

Comment (radtriste) [reply test/.default_config:1]: nope it is not.
those should be changed if we target another branch but master, like a release branch, which is not the case here

**Comment by spolti** [comment]
just waiting https://github.com/operator-framework/community-operators/pull/1876 to get merged :)
